sources/packages/backend/libs/integrations/src/services/student-assessment/student-assessment.service.ts

```diff
@@ -64,10 +68,16 @@ export class StudentAssessmentService extends RecordDataModelService<StudentAsse
         application: { student: { sinValidation: true, user: true } },
         offering: { institutionLocation: true, educationProgram: true },
       },
-      where: {
-        assessmentDate: dateEqualTo(processingDate),
-        offering: { institutionLocation: { hasIntegration: true } },
-      },
+      where: [
+        {
+          assessmentDate: dateEqualTo(processingDate),
+          offering: { institutionLocation: { hasIntegration: true } },
```

Comment:
Line 74 and 78 is repeating, can we just use it once?

Reply:
I tried, it will not work in an OR condition, we can have an AND or OR, not both simultaneously.